src/integration_fleets_tutorial.md

```diff
@@ -0,0 +1,386 @@
+    voltage: 24.0 # V
+    capacity: 40.0 # Ahr
+    charging_current: 26.4 # A
+  mechanical_system:
+    mass: 80.0 # kg
+    moment_of_inertia: 20.0 #kgm^2
+    friction_coefficient: 0.20
+  ambient_system:
+    power: 20.0 # W
+  tool_system:
+    power: 760.0 # W
+  recharge_threshold: 0.20 
+  recharge_soc: 1.0 
+  publish_fleet_state: True
+  account_for_battery_drain: True
+  task_capabilities: 
+    loop: True
+    delivery: False
+    clean: False
+    finishing_request: "nothing" # [park, charge, nothing]
+
+# DeliveryBot CONFIG =================================================================
+
+robots:
+  deliverybot1:
+    robot_config:
+      max_delay: 10.0 
+    rmf_config:
+      robot_state_update_frequency: 0.5
+      start:
+        map_name: "L1"
+        # waypoint: "charger_deliverybot1" # Optional
+        # orientation: 0.0 # Optional, radians
+        waypoint: null
+        orientation: null
+      charger:
+        waypoint: "charger_deliverybot1"
+reference_coordinates:
+  rmf: [[20.33, -3.156],
+        [8.908, -2.57],
+        [13.02, -3.601],
+        [21.93, -4.124]]
+  robot: [[59, 399],
+        [57, 172],
+        [68, 251],
+        [75, 429]]
+```
+
+  
+
+- `rmf_fleet` information about the fleet and the bots in the fleet.
+
+- `fleet_manager` prefix, username and password
+
+- `limits` maximum values for linear and angular accelerations and velocities.
+
+- `profile` footprint and vicinity.
+
+- `reversible` a flag that can enable/disable reverse traversal in the robot.
+
+- `battery_system` information about the battery
+
+- `recharge_threshold` sets a value for minimum charge below which the robot cannot operate.
+
+- `recharge_soc` the maximum level to which robots should be charged.
+
+- `task_capabilities` the capabilities of the robot
+
+- `finishing_request` can be set to `park`, `charge` or `nothing`
+
+- `robots` information about all the types of robots in this case there is only one robot i.e. delivery bot
+
+- `tinyRobot1` will have all the information required by the bot.
+
+- `max_delay` seconds before interruption occurs and replanning happens
+
+- `robot_state_update_frequency` how frequently should robot update the fleet
+
+- `map_name` name of the map
+
+- `waypoint` target location
+
+- `orientation` orientation in radians
+
+- `charger` location of the charger
+
+- `reference_coordinates` The robot and the RMF may use different coordinate system the `reference_coordinates` help in correcting them.
+
+
+## RobotClientAPI.py 
+Users can start by filling in the appropriate API inside [`RobotClientAPI.py`](https://github.com/open-rmf/rmf_demos/blob/main/rmf_demos_fleet_adapter/rmf_demos_fleet_adapter/RobotClientAPI.py#L28), which will be used by the `RobotCommandHandle` to make calls to the fleet robots. For example, if your robot uses REST API to interface with the fleet adapter, you will need to make HTTP request calls to the appropriate endpoints within these functions.
```

Comment:
Link to the template's RobotClientAPI instead. Else it might be confusing about which parts of the code they should replace. 
```suggestion
Users can fill in the appropriate API inside [`RobotClientAPI.py`](https://github.com/open-rmf/fleet_adapter_template/blob/main/fleet_adapter_template/fleet_adapter_template/RobotClientAPI.py), which will be used by the `RobotCommandHandle` to make calls to the fleet robots. For example, if your robot uses REST API to interface with the fleet adapter, you will need to make HTTP request calls to the appropriate endpoints within these functions.
```